components/Dropdown/Dropdown.jsx

```diff
@@ -212,7 +216,6 @@ const SelectedItemLabel = styled.span`
 `;
 
 const DropItemImage = styled.img`
-  float: right;
   width: ${DROPITEM_IMAGE_SIZE};
```

Comment:
if we put some margin-left to avoid this:

![image](https://user-images.githubusercontent.com/3389749/93250208-da56c300-f768-11ea-809e-d45cf852d84b.png)

with `margin-left: 8px`:

![image](https://user-images.githubusercontent.com/3389749/93250303-05411700-f769-11ea-968e-6903f2a9164d.png)

Reply:
Yeap, much better! That is a good point. 
I gonna do this change, @rapahaeru.

Reply:
Done!!
I added one more change in the `DropItemLabel` component.

Reply:
@rapahaeru, now the requested changes have been made, what is the next step?

Reply:
we're review again @alisson-suzigan